src/scraper/scraper.cpp

```diff
@@ -5434,7 +5434,91 @@ UniValue archivelog(const UniValue& params, bool fHelp)
     return UniValue(ret);
 }
 
+UniValue convergencereport(const UniValue& params, bool fHelp)
+{
+    if (fHelp || params.size() > 0 )
+        throw std::runtime_error(
+                "convergencereport\n"
+                "Display local node report of scraper convergence.\n"
+                );
+
+    // See if converged stats/contract update needed...
+    bool bConvergenceUpdateNeeded = true;
+    {
+        LOCK(cs_ConvergedScraperStatsCache);
+
+        if (GetAdjustedTime() - ConvergedScraperStatsCache.nTime < (nScraperSleep / 1000) || ConvergedScraperStatsCache.bClean)
+        {
+            bConvergenceUpdateNeeded = false;
+        }
+    }
+
+    if (bConvergenceUpdateNeeded)
+    {
+        // Don't need the output but will use the global cache, which will be updated.
+        ScraperGetSuperblockContract(false, false);
+    }
+
+    UniValue result(UniValue::VOBJ);
+
+    {
+        LOCK(cs_ConvergedScraperStatsCache);
+
+        if (!ConvergedScraperStatsCache.NewFormatSuperblock.WellFormed())
+        {
+            UniValue error(UniValue::VOBJ);
+            error.pushKV("Error:", "A convergence cannot be formed at this time. Please review the scraper.log for details.");
+
+            return error;
+        }
 
+        int64_t nConvergenceTime = ConvergedScraperStatsCache.nTime;
+
+        result.pushKV("convergence timestamp", nConvergenceTime);
+        result.pushKV("convergence datetime", DateTimeStrFormat("%x %H:%M:%S",  nConvergenceTime));
+
+        UniValue ExcludedProjects(UniValue::VARR);
+
+        for (const auto& entry : ConvergedScraperStatsCache.Convergence.vExcludedProjects)
+        {
+            ExcludedProjects.push_back(entry);
+        }
+
+        result.pushKV("excluded projects", ExcludedProjects);
```

Comment:
I've been trying to use `snake_case` for JSON object keys in new/updated RPCs so that third-party clients don't need to jump through a hoop to handle keys with spaces differently. I know this is mostly for human consumption, so it's not particularly important, but it may be good to output these uniformly just in case. What do you think?

Reply:
Yes. That is a good idea to use as a standard. I will change here, and we should stick to that in the future.